gcc/rust/ast/rust-ast-fragment.h

```diff
@@ -117,6 +122,14 @@ class Fragment
   bool is_single_fragment_of_kind (SingleASTNode::NodeType expected) const;
   void assert_single_fragment (SingleASTNode::NodeType expected) const;
 };
+
+/**
+ * This is the type for transcriber functions found in
+ * rust-macro-builtins.{h,cc}.
+ */
+using transcriber_t
```

Comment:
if we can't namespace due to a circular dep might I suggest giving it a "Macro*" name?
```suggestion
using MacroTranscriber
```

Reply:
Tried that... and saw there's already a `MacroTranscriber` struct. Went for `MacroTranscriberFunc` instead